<launch>
    <arg name="microphone_id"   default="default" />
    <arg name="model_dir"       default="$(find ros_offline_asr)/models" />
    <node name="offline_asr" pkg="ros_offline_asr" type="recognizer.py" output="screen">
        <param name="model_dir" value="$(arg model_dir)"/>
        <param name="microphone_id" value="$(arg microphone_id)"/>
    </node>
</launch>